src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+                System.out.println(addDoubleLine("     Bye. Hope to see you again soon!"));
+                System.exit(0);
+            //list
+            }else if(check.equals("list")){
+                handleList(list);
+            //done
+            }else if(check.equals("done")){
+                String taskNum = splited[1];
+                Task current = list.get(Integer.parseInt(taskNum) - 1);
+                current.markAsDone();
+                System.out.println(addDoubleLine("     Nice! I've marked this task as done:\n" + "    " + current.toString()));
+            //make a todo task
+            }else if(check.equals("delete")){
+                String taskNum = splited[1];
+                Task current = list.get(Integer.parseInt(taskNum) - 1);
+                list.remove(Integer.parseInt(taskNum) - 1);
+                System.out.println(addDoubleLine("     Noted. I've removed this task: \n" + "    " + current.toString() + "\n     Now you have " + list.size() + " tasks in the list."));
+            }else if(check.equals("todo")){
+                String description = line.replace("todo", "").trim();
+                //error handling
+                if(description.equals("")){
+                    System.out.println(addDoubleLine("    ☹ OOPS!!! The description of a todo cannot be empty."));
+                
+                }else{ //successful addition 
+                    list.add(new Todo(description));
+                    Task current = list.get(list.size()-1);
+                    printMsg(current, list.size());
+                }
+            //make an event task
+            }else if(check.equals("event")){
+                //removes the command and splits it into 2
+                String [] splitDate = line.replace("event", "").split("/at");
+                if(splitDate.length < 2){
+                    System.out.println(addDoubleLine("    ☹ OOPS!!! Events require both a description and a date \n    (e.g. event go to concert /at 13 Feb)"));
+                }else{
+                    //if it reaches here it is successful
+                    list.add(new Event(splitDate[0].trim(), splitDate[1].trim()));
+                    Task current = list.get(list.size()-1);
+                    printMsg(current, list.size());
+                
+                }
+            //make deadline task
+            }else if(check.equals("deadline")){
+                String [] splitDate = line.replace("deadline", "").split("/by");
+                if(splitDate.length < 2){
+                    System.out.println(addDoubleLine("    ☹ OOPS!!! Deadlines require both a description and a date by \n    (e.g. deadline homework3 /by tomorrow)"));
+                }else{
+                    list.add(new Deadline(splitDate[0].trim(), splitDate[1].trim()));
+                    Task current = list.get(list.size()-1);
+                    printMsg(current, list.size());
+                }
+            //error handling
+            }else{
```

Comment:
you might want to use try catch for this part, because to my understanding, if the user inputs "done hello" it will still throw error and not catch it